src/sass/_card.scss

```diff
@@ -8,6 +8,26 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    width: 100%;
```

Comment:
Logos still kinda look off for me. Have you tried cover or contain with No repeat?